Samples/Common/DockerUtils.cs

```diff
@@ -0,0 +1,408 @@
+
+namespace Microsoft.Azure.Management.Samples.Common
+{
+    /**
+     * Syncronous extension wrappers class for Docker.DotNet async methods.
+     */
+    public static class DockerExtensions
+    {
+        public static Stream PullImage(this IImageOperations operations, ImagesPullParameters parameters, AuthConfig authConfig)
+        {
+            return operations.PullImageAsync(parameters, authConfig).Result;
+        }
+
+        public static Stream PushImage(this IImageOperations operations, string name, ImagePushParameters parameters, AuthConfig authConfig)
+        {
+            return operations.PushImageAsync(name, parameters, authConfig).Result;
+        }
+
+        public static CreateContainerResponse CreateContainer(this IContainerOperations operations, CreateContainerParameters parameters)
+        {
+            return operations.CreateContainerAsync(parameters).Result;
+        }
+
+        public static IList<ImagesListResponse> ListImages(this IImageOperations operations, ImagesListParameters parameters)
+        {
+            return operations.ListImagesAsync(parameters).Result;
+        }
+
+        public static IList<ContainerListResponse> ListContainers(this IContainerOperations operations, ContainersListParameters parameters)
+        {
+            return operations.ListContainersAsync(parameters).Result;
+        }
+
+        public static void RemoveContainer(this IContainerOperations operations, string name, ContainerRemoveParameters parameters)
+        {
+            operations.RemoveContainerAsync(name, parameters).Wait();
+        }
+
+        public static void DeleteImage(this IImageOperations operations, string name, ImageDeleteParameters parameters)
+        {
+            operations.DeleteImageAsync(name, parameters).Wait();
+        }
+
+        public static CommitContainerChangesResponse CommitContainerChanges(this IMiscellaneousOperations operations, CommitContainerChangesParameters parameters)
+        {
+            return operations.CommitContainerChangesAsync(parameters).Result;
+        }
+    }
+
+    /**
+     * Utility class to be used by Azure Container Registry sample.
+     * - Creates "in memory" SSL configuration to be used by the Java Docker client
+     * - Builds a Docker client config object
+     * - Creates a new Azure virtual machine and installs Docker
+     * - Creates a Java DockerClient to be used for communicating with a Docker host/engine
+     */
+    public class DockerUtils
+    {
+        /**
+         * Instantiate a Docker client that will be used for Docker client related operations.
+         * @param azure - instance of Azure
+         * @param rgName - name of the Azure resource group to be used when creating a virtual machine
+         * @param region - region to be used when creating a virtual machine
+         * @return an instance of DockerClient
+         */
+        public static DockerClient createDockerClient(IAzure azure, String rgName, Region region)
```

Comment:
>createDockerClient [](start = 35, length = 18)

method names should start with capital letter in C#
please fix in all the methods in this file